ferrocene/doc/index/index.html

```diff
@@ -116,6 +112,10 @@ <h3>Internal Procedures</h3>
                         <h3>Technical Report</h3>
                         <p>Technical report of the qualification, written by the assessors.</p>
                     </a>
+                    <a href="requirements/index.html">
+                        <h3>Requirements</h3>
```

Comment:
Let's use the same name in `conf.py` and here.
```suggestion
                        <h3>Ferrocene Requirements</h3>
```